core/templates/dev/head/css/oppia.css

```diff
@@ -834,14 +834,27 @@ textarea {
   background-color: #eee;
 }
 
+.oppia-gallery-group {
+  width: 100vw;
```

Comment:
In each selector, could you please alphabetize the keys? It makes it easier to find them subsequently.